src/editmap.h

```diff
@@ -56,8 +56,10 @@ class editmap
         int edit_critter( Creature &critter );
         int edit_veh();
         int edit_mapgen();
+        int set_mapgen(std::string om_name, tripoint pt);
         void cleartmpmap( tinymap &tmpmap );
         int mapgen_preview( real_coords &tc, uimenu &gmenu );
+        int mapgen_set( real_coords &tc, std::string om_name);
```

Comment:
What's the difference between `mapgen_set` and `set_mapgen`? The function names don't indicate any difference (they both seem to "set" something via/for/using "mapgen"). And the parameters aren't that different either.